### PR TITLE
Update baseimages.md

### PR DESCRIPTION
**Pull request by scjane**

### CODE REVIEW

File: engine/userguide/eng-image/baseimages.md [hunk 1]
```diff
@@ -67,14 +67,14 @@ NOTE: Because Docker for Mac and Docker for Windows use a Linux VM, you must com
...
 
 ## More resources
 
-There are lots more resources available to help you write your 'Dockerfile`.
+There are lots more resources available to help you write your `Dockerfile`.
 
 * There's a [complete guide to all the instructions](../../reference/builder.md) available for use in a `Dockerfile` in the reference section.
 * To help you write a clear, readable, maintainable `Dockerfile`, we've also
-written a [`Dockerfile` Best Practices guide](dockerfile_best-practices.md).
+written a [`Dockerfile` Best Practices Guide](dockerfile_best-practices.md).
```
Comment (mstanleyjones) [inline engine/userguide/eng-image/baseimages.md]: This one is incorrect. We prefer sentence case.

Comment (scjane) [inline engine/userguide/eng-image/baseimages.md]: I'm a little confused about this. If sentence case is used, should it be "Dockerfile best practices guide"?

Comment (joaofnfernandes) [inline engine/userguide/eng-image/baseimages.md]: yes. Our capitalization rules are simple:
* Is the first word in the sentence? Capitalize
* Is it a product name? Capitalize
* Everything else, keep it lower case.

Comment (scjane) [inline engine/userguide/eng-image/baseimages.md]: @joaofnfernandes Thank you for your review and comments. I've read through the Style guide for Docker documentation and modified my commits based on the sentence-caps rule.

Comment (joaofnfernandes) [inline engine/userguide/eng-image/baseimages.md]: Shame on me for not pointing you to the style guide in the first place 😔

Comment (scjane) [inline engine/userguide/eng-image/baseimages.md]: @joaofnfernandes No, no. Shame on me for not carefully reading the style guide first before submitting any commits.

**Pull request merged**